src/periphery/contracts/misc/UiIncentiveDataProviderV3.sol

```diff
@@ -342,59 +296,6 @@ contract UiIncentiveDataProviderV3 is IUiIncentiveDataProviderV3 {
           vUserRewardsInformation
         );
       }
-
-      // stable debt token
-      IRewardsController sTokenIncentiveController = IRewardsController(
```

Comment:
have we ever had incentives on stable debt? will users be able to claim old incentives if they are eligible?